docker/malware-scanner/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM golang:1.17
```

we should use full version string (with patch, third digit) - we had some issues in the past where we didn't change anything relevant in the release but when redeploying it pulled latest version and it broke the release and we spent significant amount of time debugging that

```suggestion
FROM golang:1.17.3
```